distributions/openhab/src/main/resources/userdata/etc/shell.init.script

```diff
@@ -0,0 +1,56 @@
+//
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+// This script is run each time a shell is created.
+// You can define here closures or variables that will be available
+// in each session.
+//
+ld = { log:display $args } ;
+lde = { log:exception-display $args } ;
+la = { bundle:list -t 0 $args } ;
+ls = { service:list $args } ;
+cl = { config:list "(service.pid=$args)" } ;
+halt = { system:shutdown -h -f $args } ;
+help = { *:help $args | more } ;
+man = { help $args } ;
+log:list = { log:get ALL } ;
+service:get = { $.context getService ($.context getServiceReference $args) };
+
+env = { shell:set $args }
+edit = { shell:nano $args }
+more = { shell:less $args }
+
+//prompt = "[1m${USER}[0m@${APPLICATION}(${SUBSHELL})> "
```

Comment:
@kaikreuzer any reason this was commented, I'm debuging some problems with docker: https://community.openhab.org/t/docker-compose-vs-docker-error-in-initialization-script/34025

Reply:
Afair, it messed up our console prompt (we do not want user/application/subshell information).
If you find that we must not comment it out, but instead define "our" prompt here, we might change it to [the prompt from here](https://github.com/openhab/openhab-distro/blob/master/distributions/openhab/src/main/filtered-resources/userdata/etc/branding-ssh.properties#L15).

Reply:
Indeed I changed it in an attempt to fix but did not solve the problem but it did make the prompt ugly 😉